test/fixtures/asyncapi.yml

```diff
@@ -1,38 +1,69 @@
-asyncapi: 2.6.0
-
+asyncapi: 3.0.0
 info:
   title: Comments Service
   version: 1.0.0
   description: This service is in charge of processing all the events related to comments.
-
 servers:
   dev:
-    url: test.mosquitto.org
+    host: test.mosquitto.org
     protocol: mqtt
-
 channels:
-  comment/liked:
-    description: Updates the likes count in the database when new like is noticed.
-    publish: 
-      message:
+  commentLiked:
+    address: comment/liked
+    messages:
+      commentLiked:
         description: Message that is being sent when a comment has been liked by someone.
         payload:
-          $ref: '#/components/schemas/commentId'
-
-  comment/unliked:
-    description: Updates the likes count in the database when comment is unliked.
-    publish: 
-      operationId: commentUnliked
-      message:
+          $ref: '#/components/schemas/commentReaction'
+    description: Updates the likes count in the database when new like is noticed.
+  commentUnliked:
+    address: comment/unliked
+    messages:
+      commentUnliked:
         description: Message that is being sent when a comment has been unliked by someone.
         payload:
-          $ref: '#/components/schemas/commentId'
-
+          $ref: '#/components/schemas/commentReaction'
+    description: Updates the likes count in the database when comment is unliked.
+  commentViews:
+    address: comment/views
+    messages:
+      commentViews:
+        description: >-
+          Message that is being recived with the total number of views in a
+          comment.
+        payload:
+          $ref: '#/components/schemas/commentCount'
+    description: Gets the total number of comment views.
+operations:
+  commentLiked:
```

Comment:
think about naming operations from a perspective of the name of generated method